docs/pages/installation/index.md

```diff
@@ -63,14 +63,14 @@ If your config has not been auto-detected, you may manually locate and upload yo
 Run connectivity tests and verify the health of your Meshery system. Verify Meshery's connection to Kubernetes by clicking on your configuration `context` name. You will be notified of your connection status. You can also verify a successful connection between Meshery and its adapters by clicking on any of the available [Adapter Ports]({{ site.baseurl }}/concepts/architecture#adapter-ports).
 
 <a href="/assets/img/adapters/meshery-ui.png">
-<img style="width:450px;height=auto;" src="/assets/img/adapters/meshery-ui.png" />
+<img style="width:550px;height=auto;" src="/assets/img/adapters/meshery-ui.png" />
```

Comment:
The static sizing concerns me a that responsiveness might be a bit fragile.

If there's a more clever approach that is simple to implement, I encourage us to take moment to consider it. If not, let's press forward.

Reply:
This is just to take a default value when the page is opened, i have added the responsiveness in the css file. That is the last commit @leecalcote

Reply:
Thanks @warunicorn19 👍